jetty-util/pom.xml

```diff
@@ -91,6 +91,10 @@
       <scope>provided</scope>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>commons-pool</groupId>
+      <artifactId>commons-pool</artifactId>
```

Comment:
we try to minimise dependencies with jetty so it would be better not to import `commons-pool` here, so will probably have to implement the object pool from scratch similar to the previous implementation of `DeflaterPool`